# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Aron Gergely, 2022
# Aron Gergely, 2022
# Mariann Urban, 2022
# Mariann Urban, 2022
msgid ""
msgstr ""
"Project-Id-Version: KoboToolbox\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-10-09 17:21+0000\n"
"PO-Revision-Date: 2016-03-14 19:56+0000\n"
"Last-Translator: Mariann Urban, 2022\n"
"Language-Team: Hungarian (http://app.transifex.com/kobotoolbox/kobotoolbox/language/hu/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: hu\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: hub/templates/admin/loginas/change_form.html:7
msgid "Log in as user"
msgstr "Bejelentkezés felhasználóként"

#: kobo/apps/accounts/forms.py:53
msgid "Male"
msgstr "Férfi"

#: kobo/apps/accounts/forms.py:54
msgid "Female"
msgstr "Nő"

#: kobo/apps/accounts/forms.py:55 kobo/static_lists.py:37
msgid "Other"
msgstr "Egyéb"

#: kobo/apps/accounts/forms.py:77
msgid "Password confirmation"
msgstr "Jelszó megerősítése"

#: kobo/apps/accounts/forms.py:183
msgid "You must type the same password each time."
msgstr "Ugyanazt a jelszót kell beírnia minden alkalommal."

#: kobo/apps/accounts/mfa/forms.py:57
msgid "Enter the ##token length##-character token"
msgstr "Írja be a ##token length## karakterből álló tokent"

#: kobo/apps/accounts/mfa/forms.py:69
msgid "Your token is invalid"
msgstr "Érvénytelen token"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:21
msgid "Please enter your verification token"
msgstr "Adja meg az ellenőrző kódját"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:25
msgid ""
"Please enter the verification token displayed by your authenticator app."
msgstr "Adja meg a hitelesítő alkalmazás által kijelzett ellenőrző kódot."

#: kobo/apps/accounts/mfa/templates/mfa_token.html:32
msgid "Problems with the token"
msgstr "Problémák vannak a kóddal"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:37
msgid "Continue"
msgstr "Tovább"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:45
msgid "Verification issues"
msgstr "Ellenőrzési problémák"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:53
msgid "Back"
msgstr "Vissza"

#: kobo/apps/accounts/templates/account/email/email_confirmation_message.txt:3
msgid ""
"To confirm the new email address associated with your KoboToolbox account, "
"please follow the link below."
msgstr "A KoboToolbox-fiókjához rendelt e-mail-cím nyugtázásához kövesse az alábbi hivatkozást."

#: kobo/apps/accounts/templates/account/email/email_confirmation_message.txt:7
msgid ""
"This link will expire in one hour. To request a new verification link, "
"please go to your account settings and re-enter your new email address:"
msgstr "Ez a hivatkozás egy óra múlva lejár. Egy új ellenőrző hivatkozásért adja meg ismét új e-mail-címét a fiókbeállításoknál:"

#: kobo/apps/accounts/templates/account/email/email_confirmation_message.txt:10
#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:29
#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:15
#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:15
#: kobo/apps/hook/templates/reports/failures_email_body.html:37
#: kobo/apps/hook/templates/reports/failures_email_body.txt:29
msgid "Best,"
msgstr "Üdvözlettel,"

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:10
msgid "Thanks for signing up with KoboToolbox!"
msgstr "Köszönjük, hogy regisztrált a KoboToolbox alkalmazásra!"

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:12
msgid ""
"Confirming your account will give you access to KoboToolbox applications. "
"Please visit the following URL to finish activation of your new account."
msgstr "Ha megerősíti a fiókját, hozzáférést kap a KoboToolbox alkalmazásokhoz. Új fiókjának aktiválását a következő URL-címre látogatva fejezheti be."

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:16
msgid "Your username is: "
msgstr "A felhasználóneve: "

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:25
msgid ""
"For help getting started, check out the KoboToolbox user documentation: "
"https://support.kobotoolbox.com "
msgstr "Az első lépésekhez a KoboToolbox felhasználói dokumentációjában talál segítséget: https://support.kobotoolbox.com "

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:27
msgid ""
"You can also join the KoboToolbox community forum to ask questions, share "
"solutions, and chat with thousands of users: "
"https://community.kobotoolbox.org"
msgstr "Csatlakozhat a KoboToolbox közösségi fórumához is, ahol feltehet kérdéseket, megoszthatja megoldásait és több ezer felhasználóval cseveghet: https://community.kobotoolbox.org"

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_subject.txt:10
msgid "Activate your KoboToolbox Account"
msgstr "Aktiválja KoboToolbox-fiókját"

#: kobo/apps/accounts/templates/account/email/email_confirmation_subject.txt:3
msgid "KoboToolbox account email address verification"
msgstr "KoboToolbox-fiókhoz tartozó e-mail-cím ellenőrzése"

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:4
#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:5
msgid "Hello from KoboToolbox,"
msgstr "Üdvözlet a KoboToolbox csapatától!"

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:6
msgid "We’ve received a password reset request for your user account."
msgstr "Jelszó-visszaállítási kérelmet kaptunk a felhasználói fiókjához."

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:8
msgid "To reset your password, click the link below."
msgstr "A jelszó visszaállításához kattintson az alábbi hivatkozásra."

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:11
#, python-format
msgid "Your username is %(username)s."
msgstr "A felhasználóneve %(username)s."

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:13
#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:9
msgid ""
"If you did not request a password reset, you can safely ignore this email."
msgstr "Amennyiben nem kért jelszó-visszaállítást, nyugodtan figyelmen kívül hagyhatja ezt az e-mailt."

#: kobo/apps/accounts/templates/account/email/password_reset_key_subject.txt:3
msgid "KoboToolbox Password Reset"
msgstr "KoboToolbox-jelszó visszaállítása"

#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:6
msgid ""
"You are receiving this email because you or someone else has requested a "
"password for their KoboToolbox account."
msgstr "Azért kapta ezt a levelet, mert Ön vagy valaki más kért egy jelszót ehhez a KoboToolbox-fiókhoz."

#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:7
#, python-format
msgid ""
"However, we do not have any record of an account with email %(email)s in our"
" database."
msgstr "Azonban nem található %(email)s e-mail-címmel fiók az adatbázisunkban."

#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:11
msgid ""
"If it was you, please try another email address you might have used to "
"create your account, or double check that your account was not registered on"
" a different KoboToolbox server."
msgstr "Amennyiben Ön volt a kérelmező, próbáljon ki egy másik e-mail-címet, amelyet a fiók létrehozásához használhatott, vagy ellenőrizze, hogy a fiókja nem egy másik KoboToolbox-kiszolgálón volt-e regisztrálva."

#: kobo/apps/accounts/templates/account/email/unknown_account_subject.txt:3
msgid "KoboToolbox Password Reset Unsuccessful"
msgstr "KoboToolbox-jelszó helyreállítása sikertelen"

#: kobo/apps/accounts/templates/account/email_confirm.html:15
msgid "Confirm E-mail Address"
msgstr "E-mail-cím megerősítése"

#: kobo/apps/accounts/templates/account/email_confirm.html:19
#, python-format
msgid ""
"Please confirm that <a href=\"mailto:%(email)s\">%(email)s</a> is an e-mail "
"address for user %(user_display)s."
msgstr "Kérjük ellenőrizze, hogy a(z) <a href=\"mailto:%(email)s\">%(email)s</a> valóban a(z) %(user_display)s felhasználó e-mail-cím."

#: kobo/apps/accounts/templates/account/email_confirm.html:26
msgid "Confirm"
msgstr "Megerősítés"

#: kobo/apps/accounts/templates/account/email_confirm.html:30
msgid "Activation failed"
msgstr "Sikertelen aktiválás"

#: kobo/apps/accounts/templates/account/email_confirm.html:33
msgid ""
"Sorry, we couldn't activate your account. The link we sent you has expired."
msgstr "Sajnáljuk, fiókját nem tudtuk aktiválni. A hivatkozás lejárt."

#: kobo/apps/accounts/templates/account/email_confirm.html:39
msgid ""
"This e-mail confirmation link expired or is invalid.\n"
"      If you need to request a new link to verify your email address change,\n"
"      please go to your account settings page and enter your email address again."
msgstr "Az e-mail-cím megerősítésére szolgáló hivatkozás lejárt vagy érvénytelen.\n     Amennyiben új ellenőrző hivatkozásra van szüksége, \n     adja meg ismét az e-mail-címét a fiókbeállításoknál."

#: kobo/apps/accounts/templates/account/login.html:29
msgid "Login"
msgstr "Bejelentkezés"

#: kobo/apps/accounts/templates/account/login.html:36
#: kobo/apps/accounts/templates/account/signup.html:14
msgid "Create an account"
msgstr "Fiók létrehozása"

#: kobo/apps/accounts/templates/account/login.html:41
msgid "Forgot password?"
msgstr "Elfelejtett jelszó?"

#: kobo/apps/accounts/templates/account/login.html:50
msgid "Log in using SSO"
msgstr "Belépés SSO használatával"

#: kobo/apps/accounts/templates/account/login.html:52
#: kobo/apps/accounts/templates/account/signup.html:61
msgid ""
"If you set up Single Sign On in your account settings, you will only be able"
" to log in through this method."
msgstr "Amennyiben egyszeri bejelentkezést (SSO) állított be a fiókbeállításoknál, csak ezzel a módszerrel fog tudni bejelentkezni."

#: kobo/apps/accounts/templates/account/password_reset.html:10
msgid "Password Reset"
msgstr "Jelszó-visszaállítás"

#: kobo/apps/accounts/templates/account/password_reset.html:19
msgid "Reset Password"
msgstr "Jelszó-visszaállítás"

#: kobo/apps/accounts/templates/account/password_reset_complete.html:11
#: kobo/apps/accounts/templates/account/password_reset_from_key_done.html:11
msgid "Password reset successfully"
msgstr "Sikeres jelszó-visszaállítás"

#: kobo/apps/accounts/templates/account/password_reset_complete.html:14
#: kobo/apps/accounts/templates/account/password_reset_from_key_done.html:14
#: kobo/apps/accounts/templates/socialaccount/login.html:34
msgid "Log in"
msgstr "Bejelentkezés"

#: kobo/apps/accounts/templates/account/password_reset_done.html:12
msgid ""
"Your request has been received. If the email address you entered corresponds"
" to an account on this service, a message has been sent with password reset "
"instructions. If you do not receive an email, your account might be "
"registered under a different address, or you might have entered your address"
" incorrectly."
msgstr "Kérését megkaptuk. Amennyiben a megadott e-mail-címhez tartozik egy fiók ezen a szolgáltatáson, egy üzenetet küldtünk a jelszó-visszaállítási utasításokkal. Amennyiben nem kapott e-mailt, fiókjához más e-mail-cím tartozhat, esetleg e-mail-címét tévesen adta meg."

#: kobo/apps/accounts/templates/account/password_reset_from_key.html:13
msgid "Password reset failed"
msgstr "Sikertelen jelszó-helyreállítás"

#: kobo/apps/accounts/templates/account/password_reset_from_key.html:25
msgid "Submit"
msgstr "Elküld"

#: kobo/apps/accounts/templates/account/signup.html:45
msgid "Create Account"
msgstr "Fiók létrehozása"

#: kobo/apps/accounts/templates/account/signup.html:49
#: kobo/apps/accounts/templates/socialaccount/login.html:39
msgid "or"
msgstr "vagy"

#: kobo/apps/accounts/templates/account/signup.html:49
msgid "login"
msgstr "bejelentkezés"

#: kobo/apps/accounts/templates/account/signup.html:59
msgid "Register using SSO"
msgstr "Regisztráció SSO használatával"

#: kobo/apps/accounts/templates/account/signup.html:73
msgid "Register with SSO"
msgstr "Regisztráció SSO-val"

#: kobo/apps/accounts/templates/account/verification_sent.html:8
msgid "Confirm your email address"
msgstr "Erősítse meg az e-mail-címét"

#: kobo/apps/accounts/templates/account/verification_sent.html:11
msgid "Please click the activation link in the email just sent to you."
msgstr "Kattintson az aktiváló hivatkozásra, amely az Önnek imént elküldött e-mailben van."

#: kobo/apps/accounts/templates/socialaccount/login.html:14
msgid "Set up Single Sign On"
msgstr "Egyszeri bejelentkezés (SSO) beállítása"

#: kobo/apps/accounts/templates/socialaccount/login.html:16
#, python-format
msgid ""
"Click the button below to connect your KoboToolbox account with your "
"%(appname)s account."
msgstr "Kattintson az alábbi gombra a KoboToolbox-fiókjának a(z) %(appname)s fiókkal való összekapcsolásához."

#: kobo/apps/accounts/templates/socialaccount/login.html:18
msgid "This will allow you to use single sign-on (SSO) in the future."
msgstr "Ezáltal a jövőben használhatja az egyszeri bejelentkezést (SSO)."

#: kobo/apps/accounts/templates/socialaccount/login.html:23
#, python-format
msgid "Connect with %(appname)s"
msgstr "Összekapcsolás ezzel a fiókkal: %(appname)s"

#: kobo/apps/accounts/templates/socialaccount/login.html:27
msgid "Log in with "
msgstr "Belépés ezzel: "

#: kobo/apps/accounts/templates/socialaccount/login.html:29
#, python-format
msgid "Log in to KoboToolbox with your %(appname)s account."
msgstr "Belépés a KoboToolbox alkalmazásba a(z) %(appname)s-fiókjával."

#: kobo/apps/accounts/templates/socialaccount/login.html:39
msgid "go back"
msgstr "vissza"

#: kobo/apps/accounts/templates/socialaccount/signup.html:11
#, python-format
msgid "Welcome to KoboToolbox, %(username)s!"
msgstr "Üdvözli a KoboToolbox, kedves %(username)s!"

#: kobo/apps/accounts/templates/socialaccount/signup.html:13
msgid "Welcome to KoboToolbox!"
msgstr "Üdvözli a KoboToolbox!"

#: kobo/apps/accounts/templates/socialaccount/signup.html:18
#, python-format
msgid "You are about to use your %(appname)s account to login to KoboToolbox."
msgstr "A(z) %(appname)s-fiókjával készül a KoboToolbox alkalmazásba bejelentkezni."

#: kobo/apps/accounts/templates/socialaccount/signup.html:18
msgid "As a final step, please complete the following form"
msgstr "Utolsó lépésként töltse ki a következő űrlapot"

#: kobo/apps/accounts/templates/socialaccount/signup.html:43
msgid "Save and Access"
msgstr "Mentés és hozzáférés"

#: kobo/apps/accounts/validators.py:7
msgid ""
"Usernames must be between 2 and 30 characters in length, and may only "
"consist of lowercase letters, numbers, and underscores, where the first "
"character must be a letter."
msgstr "A felhasználónév hosszúságának 2 és 30 karakter között kell lennie, csak kisbetűt, számokat és aláhúzásjelet tartalmazhat, és az első karakter csak betű lehet."

#: kobo/apps/hook/serializers/v2/hook.py:46
msgid "Invalid scheme"
msgstr "Érvénytelen séma"

#: kobo/apps/hook/serializers/v2/hook.py:49
msgid "Unsecured endpoint is not allowed"
msgstr "Biztonsági szempontból nem védett végpont nem engedélyezett"

#: kobo/apps/hook/serializers/v2/hook.py:60
msgid "Invalid JSON"
msgstr "Érvénytelen JSON"

#: kobo/apps/hook/serializers/v2/hook.py:70
msgid "Can be used only with JSON submission format"
msgstr "Csak JSON űrlapválasz formátumban használható"

#: kobo/apps/hook/tasks.py:148
msgid "REST Services Failure Report"
msgstr "REST szolgáltatások hibajelentése"

#: kobo/apps/hook/templates/reports/failures_email_body.html:3
#: kobo/apps/hook/templates/reports/failures_email_body.txt:3
msgid "Project name"
msgstr "Projektnév"

#: kobo/apps/hook/templates/reports/failures_email_body.html:4
#: kobo/apps/hook/templates/reports/failures_email_body.txt:4
msgid "Hook"
msgstr "Hook"

#: kobo/apps/hook/templates/reports/failures_email_body.html:5
#: kobo/apps/hook/templates/reports/failures_email_body.txt:5
msgid "Submission"
msgstr "Űrlapválasz"

#: kobo/apps/hook/templates/reports/failures_email_body.html:6
#: kobo/apps/hook/templates/reports/failures_email_body.txt:6
msgid "Status code"
msgstr "Állapotkód"

#: kobo/apps/hook/templates/reports/failures_email_body.html:7
#: kobo/apps/hook/templates/reports/failures_email_body.txt:7
msgid "Message"
msgstr "Üzenet"

#: kobo/apps/hook/templates/reports/failures_email_body.html:8
#: kobo/apps/hook/templates/reports/failures_email_body.txt:8
msgid "Date"
msgstr "Dátum"

#: kobo/apps/hook/templates/reports/failures_email_body.html:10
#: kobo/apps/hook/templates/reports/failures_email_body.txt:10
msgid "Dear"
msgstr "Kedves"

#: kobo/apps/hook/templates/reports/failures_email_body.html:11
#: kobo/apps/hook/templates/reports/failures_email_body.txt:12
msgid "Some submissions could not be sent to their external endpoints."
msgstr "Egyes űrlapválaszokat nem sikerült elküldeni a külső végpontoknak."

#: kobo/apps/hook/views/v2/hook.py:174
msgid "Task successfully scheduled"
msgstr "A feladat ütemezése sikeres"

#: kobo/apps/hook/views/v2/hook.py:203
msgid "No data to retry"
msgstr "Nincs adat újrapróbálkozásra"

#: kobo/apps/hook/views/v2/hook.py:206
msgid "Can not retry on disabled hooks"
msgstr "Nem próbálkozhat újra letiltott hookokon"

#: kobo/apps/hook/views/v2/hook_log.py:121
msgid "An error has occurred when sending the data. Please try again later."
msgstr "Az adat küldésekor hiba lépett fel. Kérjük, később próbálja meg újra."

#: kobo/apps/hook/views/v2/hook_log.py:125
msgid "Data is being or has already been processed"
msgstr "Az adat már fel van dolgozva vagy feldolgozás alatt áll"

#: kobo/apps/hook/views/v2/hook_signal.py:54
#: kpi/deployment_backends/base_backend.py:449
#: kpi/deployment_backends/base_backend.py:456
msgid "A positive integer is required."
msgstr "Egy pozitív egész szám szükséges."

#: kobo/apps/hook/views/v2/hook_signal.py:71
msgid ""
"We got and saved your data, but may not have fully processed it. You should "
"not try to resubmit."
msgstr "Megkaptuk és elmentettük az adatait, de lehet, hogy még nincsenek teljesen feldolgozva. Ne próbálja meg újra beküldeni."

#: kobo/apps/hook/views/v2/hook_signal.py:80
msgid "Your data for instance {} has been already submitted."
msgstr "A(z) {} példányra már be lettek küldve az adatai."

#: kobo/apps/languages/templates/admin/languages_change_list.html:8
msgid "Import"
msgstr "Importálás"

#: kobo/apps/reports/report_data.py:136
msgid "`{}` not found."
msgstr "`{}` nem található."

#: kobo/apps/reports/report_data.py:141
msgid "`{}` is not a select one question."
msgstr "`{}` nem feleletválasztós kérdés."

#: kobo/settings/base.py:243
msgid ""
"If you cannot access your authenticator app, please enter one of your backup"
" codes instead. If you cannot access those either, then you will need to "
"request assistance by contacting [##support email##](mailto:##support "
"email##)."
msgstr "Ha nem tud hozzáférni a hitelesítő alkalmazásához, helyette az egyik biztonsági kódját adja meg. Ha azokhoz sem tud hozzáférni, akkor kérjen segítséget a következő e-mail címen: [##support email##](mailto:##support email##)."

#: kobo/settings/base.py:451
msgid ""
"The password must be at least 10 characters long and contain 3 or more of "
"the following: uppercase letters, lowercase letters, numbers, and special "
"characters. It cannot be similar to your name, username, or email address."
msgstr "A jelszónak legalább 10 karakter hosszúságúnak kell lennie és a következők közül legalább 3-at kell tartalmaznia: nagybetűk, kisbetűk, számok és speciális karakterek. Nem lehet hasonló a nevéhez, felhasználói nevéhez vagy az e-mail címéhez."

#: kobo/static_lists.py:13
msgid "Public Administration"
msgstr "Közigazgatás"

#: kobo/static_lists.py:14
msgid "Arts, Entertainment, and Recreation"
msgstr "Művészetek, szórakoztatás és szabadidő"

#: kobo/static_lists.py:15
msgid "Educational Services / Higher Education"
msgstr "Oktatási szolgáltatások / felsőoktatás"

#: kobo/static_lists.py:16
msgid "Health Services / Public Health"
msgstr "Egészségügyi szolgáltatások / közegészségügy"

#: kobo/static_lists.py:17
msgid "Finance and Insurance"
msgstr "Pénzügy és biztosítás"

#: kobo/static_lists.py:18
msgid "Information / Media"
msgstr "Tájékoztatás / média"

#: kobo/static_lists.py:19
msgid "Economic/Social Development"
msgstr "Gazdasági/szociális fejlesztés"

#: kobo/static_lists.py:20
msgid "Security / Police / Peacekeeping"
msgstr "Közbiztonság / Rendfenntartás / Békefenntartás"

#: kobo/static_lists.py:21
msgid "Disarmament & Demobilization"
msgstr "Leszerelés és feloszlatás"

#: kobo/static_lists.py:22
msgid "Environment"
msgstr "Környezetvédelem"

#: kobo/static_lists.py:23
msgid "Private sector"
msgstr "Magánszektor"

#: kobo/static_lists.py:24
msgid "Humanitarian - Coordination / Information Management"
msgstr "Humanitárius – Koordináció/információkezelés"

#: kobo/static_lists.py:25
msgid "Humanitarian - Multiple Clusters"
msgstr "Humanitárius – Több klaszter"

#: kobo/static_lists.py:26
msgid "Humanitarian - Camp Management & Coordination"
msgstr "Humanitárius – Táborirányítás és -koordináció"

#: kobo/static_lists.py:27
msgid "Humanitarian - Early Recovery"
msgstr "Humanitárius – Korai helyreállítás"

#: kobo/static_lists.py:28
msgid "Humanitarian - Education"
msgstr "Humanitárius – Oktatás"

#: kobo/static_lists.py:29
msgid "Humanitarian - Emergency Shelter"
msgstr "Humanitárius – Menedékbiztosítás"

#: kobo/static_lists.py:30
msgid "Humanitarian - Emergency Telecoms"
msgstr "Humanitárius – Vészhelyzeti távközlés"

#: kobo/static_lists.py:31
msgid "Humanitarian - Food Security"
msgstr "Humanitárius – Élelmiszerbiztonság"

#: kobo/static_lists.py:32
msgid "Humanitarian - Health"
msgstr "Humanitárius – Egészségvédelem"

#: kobo/static_lists.py:33
msgid "Humanitarian - Logistics"
msgstr "Humanitárius – Logisztika"

#: kobo/static_lists.py:34
msgid "Humanitarian - Nutrition"
msgstr "Humanitárius – Élelmezés"

#: kobo/static_lists.py:35
msgid "Humanitarian - Protection"
msgstr "Humanitárius – Védelem"

#: kobo/static_lists.py:36
msgid "Humanitarian - Sanitation, Water & Hygiene"
msgstr "Humanitárius – Szennyvízelvezetés, vízellátás és higiénia"

#: kobo/static_lists.py:344 kobo/static_lists.py:355
msgid "Sector"
msgstr "Ágazat"

#: kobo/static_lists.py:345 kobo/static_lists.py:359
msgid "Country"
msgstr "Ország"

#: kobo/static_lists.py:346
msgid "Operational purpose of data"
msgstr "Az adatkezelés célja"

#: kobo/static_lists.py:347
msgid "Does this project collect personally identifiable information?"
msgstr "Gyűjt-e ez a projekt személyes adatokat?"

#: kobo/static_lists.py:348
msgid "Description"
msgstr "Leírás"

#: kobo/static_lists.py:352 kpi/password_validation.py:109
msgid "Full name"
msgstr "Teljes név"

#: kobo/static_lists.py:353
msgid "Organization"
msgstr "Szervezet"

#: kobo/static_lists.py:354
msgid "Organization website"
msgstr "Szervezet webhelye"

#: kobo/static_lists.py:356
msgid "Gender"
msgstr "Nem"

#: kobo/static_lists.py:357
msgid "Bio"
msgstr "Önéletrajz"

#: kobo/static_lists.py:358
msgid "City"
msgstr "Város"

#: kobo/static_lists.py:360
msgid "Twitter"
msgstr "Twitter"

#: kobo/static_lists.py:361
msgid "LinkedIn"
msgstr "LinkedIn"

#: kobo/static_lists.py:362
msgid "Instagram"
msgstr "Instagram"

#: kpi/authentication.py:59
msgid "Invalid username/password"
msgstr "Érvénytelen felhasználónév/jelszó"

#: kpi/deployment_backends/base_backend.py:378
msgid ""
"This param is not implemented. Use `count` property of the response instead."
msgstr "Ez a paraméter nincs implementálva. Használja ehelyett a válasz `count` tulajdonságát."

#: kpi/deployment_backends/base_backend.py:389
msgid "This param is not supported in `XML` format"
msgstr "Ez a paraméter nem támogatott `XML` formátumban"

#: kpi/deployment_backends/base_backend.py:394
msgid "This is not supported in `XML` format"
msgstr "Nem támogatott `XML` formátumban"

#: kpi/deployment_backends/base_backend.py:414
#: kpi/deployment_backends/base_backend.py:442
#: kpi/deployment_backends/base_backend.py:464 kpi/views/v2/data.py:522
msgid "Value must be valid JSON."
msgstr "Az értéknek érvényes JSON-nak kell lennie."

#: kpi/deployment_backends/base_backend.py:419
msgid "Value must be a list."
msgstr "Az értéknek listának kell lennie."

#: kpi/deployment_backends/kobocat_backend.py:180
#: kpi/deployment_backends/mock_backend.py:100
msgid "No submissions match the given `submission_ids`"
msgstr "Nincs olyan űrlapválasz, ahol ez a `submission_ids` értéke"

#: kpi/deployment_backends/kobocat_backend.py:551
msgid "Your submission XML is malformed."
msgstr "A beküldött űrlapválasz XML formátuma hibás."

#: kpi/deployment_backends/kobocat_backend.py:558
msgid "Your submission XML is missing critical elements."
msgstr "A beküldött űrlapválasz XML-fájljából hiányoznak kritikus elemek."

#: kpi/deployment_backends/kobocat_backend.py:571
msgid ""
"The submission you attempted to edit could not be found, or you do not have "
"access to it."
msgstr "Az űrlapválasz, amelyet szerkeszteni próbált, nem található vagy nincs hozzáférése."

#: kpi/deployment_backends/kobocat_backend.py:1596
msgid "KoBoCAT returned an unexpected response: {}"
msgstr "A KoBoCAT váratlan választ adott: {}"

#: kpi/deployment_backends/kobocat_backend.py:1648
msgid "Something went wrong"
msgstr "Valamilyen hiba történt"

#: kpi/exceptions.py:58
msgid "Must call `asset.connect_deployment()` first"
msgstr "Először az `asset.connect_deployment()` eljárást kell kezdeményezni"

#: kpi/exceptions.py:74
msgid ""
"Your access is restricted. Please reclaim your access by changing your "
"password at ##koboform_url##/accounts/password/reset/."
msgstr "A hozzáférése korlátozva van. A hozzáférését a jelszava megváltoztatásával szerezheti vissza: ##koboform_url##/accounts/password/reset/."

#: kpi/exceptions.py:83
msgid "Invalid search. Please try again"
msgstr "Érvénytelen keresés. Próbálja meg újra"

#: kpi/exceptions.py:105
msgid "Invalid payload for bulk updating of submissions"
msgstr "Érvénytelen adatcsomag az űrlapválaszok tömeges frissítéséhez"

#: kpi/exceptions.py:112
msgid "An error occurred trying to bulk update the submissions."
msgstr "Az űrlapválaszok tömeges frissítésénél hiba lépett fel."

#: kpi/exceptions.py:136
msgid "An error occurred trying to duplicate the submission"
msgstr "Hiba lépett fel az űrlapválasz-másolat készítése közben"

#: kpi/exceptions.py:150
msgid "The specified object has not been deployed"
msgstr "A megadott eszköz még nincs élesítve"

#: kpi/exceptions.py:159
msgid "Bad syntax"
msgstr "Rossz szintaxis"

#: kpi/exceptions.py:164
msgid "Not supported field lookup"
msgstr "Nem támogatott mezőkeresés"

#: kpi/exceptions.py:175
msgid "Your query is too short"
msgstr "A lekérdezése túl rövid"

#: kpi/fields/jsonschema_form_field.py:23
#: kpi/fields/jsonschema_form_field.py:26
msgid "Enter valid JSON."
msgstr "Adjon meg érvényes JSON-fájlt."

#: kpi/fields/jsonschema_form_field.py:158
msgid "`##place_holder##` field cannot be hidden."
msgstr "A(z) `##place_holder##` mező nem lehet rejtett."

#: kpi/fields/writable_json.py:30
#, python-brace-format
msgid "Unable to parse JSON: {error}"
msgstr "Nem sikerült értelmezni az alábbi JSON-t: {error}"

#: kpi/mixins/mfa.py:24
msgid "Multi-factor authentication is enabled for this account. "
msgstr "Ehhez a fiókhoz a többfaktoros hitelesítés engedélyezve van. "

#: kpi/models/asset.py:249
msgid "Can view asset"
msgstr "Az eszköz megtekinthető"

#: kpi/models/asset.py:250
msgid "Can discover asset in public lists"
msgstr "Az eszköz felfedezhető nyilvános listákban"

#: kpi/models/asset.py:251
msgid "Can manage all aspects of asset"
msgstr "Az eszköz minden tulajdonsága elérhető"

#: kpi/models/asset.py:253
msgid "Can submit data to asset"
msgstr "Az adatok beküldhetők az eszközhöz"

#: kpi/models/asset.py:254
msgid "Can view submitted data for asset"
msgstr "Az eszközhöz beküldött adatok megtekinthetők"

#: kpi/models/asset.py:255
msgid ""
"Can make partial actions on submitted data for asset for specific users"
msgstr "Részleges műveletek végrehajthatók a beküldött adatokkal az eszközön, meghatározott felhasználók számára"

#: kpi/models/asset.py:258
msgid "Can modify submitted data for asset"
msgstr "Az eszközhöz beküldött adatok módosíthatók"

#: kpi/models/asset.py:259
msgid "Can delete submitted data for asset"
msgstr "Az eszközhöz beküldött adatok törölhetők"

#: kpi/models/asset.py:260
msgid "Can validate submitted data asset"
msgstr "Az eszközhöz beküldött adatok érvényesíthetők"

#: kpi/models/asset.py:283
msgid "project"
msgstr "projekt"

#: kpi/models/asset.py:283
msgid "form"
msgstr "űrlap"

#: kpi/models/asset.py:285
msgid "template"
msgstr "sablon"

#: kpi/models/asset.py:286
msgid "block"
msgstr "blokk"

#: kpi/models/asset.py:287
msgid "question"
msgstr "kérdés"

#: kpi/models/asset.py:288
msgid "text"
msgstr "szöveg"

#: kpi/models/asset.py:289
msgid "empty"
msgstr "üres"

#: kpi/models/asset.py:290
msgid "collection"
msgstr "gyűjtemény"

#: kpi/models/asset.py:297
msgid "View ##asset_type_label##"
msgstr "##asset_type_label## megtekintése"

#: kpi/models/asset.py:298
msgid "Edit ##asset_type_label##"
msgstr "##asset_type_label## szerkesztése"

#: kpi/models/asset.py:299
msgid "Discover ##asset_type_label##"
msgstr "##asset_type_label## felfedezése"

#: kpi/models/asset.py:300
msgid "Manage ##asset_type_label##"
msgstr "##asset_type_label## kezelése"

#: kpi/models/asset.py:301
msgid "Add submissions"
msgstr "Űrlapválaszok hozzáadása"

#: kpi/models/asset.py:302
msgid "View submissions"
msgstr "Űrlapválaszok megtekintése"

#: kpi/models/asset.py:305
msgid "Act on submissions only from specific users"
msgstr "Végrehajtás csak meghatározott felhasználók által beküldött űrlapválaszokon"

#: kpi/models/asset.py:308
msgid "View submissions only from specific users"
msgstr "Csak meghatározott felhasználóktól érkezett űrlapválaszok megtekintése"

#: kpi/models/asset.py:311
msgid "Edit submissions only from specific users"
msgstr "Csak meghatározott felhasználóktól érkezett űrlapválaszok szerkesztése"

#: kpi/models/asset.py:314
msgid "Delete submissions only from specific users"
msgstr "Csak meghatározott felhasználóktól érkezett űrlapválaszok törlése"

#: kpi/models/asset.py:317
msgid "Validate submissions only from specific users"
msgstr "Csak meghatározott felhasználóktól érkezett űrlapválaszok érvényesítése"

#: kpi/models/asset.py:320
msgid "Edit submissions"
msgstr "Űrlapválaszok szerkesztése"

#: kpi/models/asset.py:321
msgid "Delete submissions"
msgstr "Űrlapválaszok törlése"

#: kpi/models/asset.py:322
msgid "Validate submissions"
msgstr "Űrlapválaszok érvényesítése"

#: kpi/models/asset.py:558
msgid "Only partial permissions for submissions are supported"
msgstr "Az űrlapválaszokhoz csak részleges engedélyek támogatottak"

#: kpi/models/asset.py:1256
msgid "Can not assign '{}' permission to owner"
msgstr "Nem lehet „{}” jogosultságot hozzárendelni a tulajdonoshoz"

#: kpi/models/asset.py:1260
msgid "Can not assign '{}' permission. Partial permissions are missing."
msgstr "Nem lehet „{}” jogosultságot hozzárendelni. Részleges jogosultságok hiányoznak."

#: kpi/models/authorized_application.py:40
msgid "Invalid token."
msgstr "Érvénytelen token."

#: kpi/models/import_export_task.py:142
msgid "Cannot access data"
msgstr "Az adatok nem hozzáférhetőek"

#: kpi/models/import_export_task.py:578
msgid "This data does not exist or you do not have access to it"
msgstr "Ez az adat nem létezik vagy nincs hozzáférése"

#: kpi/password_validation.py:95
msgid "You cannot reuse your last password."
msgstr "Nem használhatja újra az utolsó jelszavát."

#: kpi/password_validation.py:103
msgid "Your password cannot be the same as your previous one."
msgstr "A jelszava nem lehet ugyanaz, mint az előző volt."

#: kpi/password_validation.py:110
msgid "Organization name"
msgstr "Szervezet neve"

#: kpi/password_validation.py:170
msgid ""
"The password must contain at least ##number of rules## different kinds of "
"characters."
msgstr "A jelszónak legalább ##number of rules## különböző típusú karaktert kell tartalmaznia."

#: kpi/serializers/current_user.py:150
msgid "Incorrect current password."
msgstr "Hibás jelenlegi jelszó."

#: kpi/serializers/current_user.py:170
msgid "`current_password` and `new_password` must both be"
msgstr "Mind a `current_password`, mind a `new_password` szükséges"

#: kpi/serializers/current_user.py:197
#: kpi/serializers/v2/asset_permission_assignment.py:287
msgid "This field may not be blank."
msgstr "Ez a mező nem lehet üres."

#: kpi/serializers/v2/asset.py:130
#, python-format
msgid "%(count)d project has been undeleted"
msgstr "%(count)d projekt törlése vissza lett vonva"

#: kpi/serializers/v2/asset.py:136
#, python-format
msgid "%(count)d project has been deleted"
msgstr "%(count)d projekt törölve lett"

#: kpi/serializers/v2/asset.py:143
#, python-format
msgid "%(count)d project has been unarchived"
msgstr "%(count)d projekt archiválása vissza lett vonva"

#: kpi/serializers/v2/asset.py:149
#, python-format
msgid "%(count)d project has been archived"
msgstr "%(count)d projekt archiválva lett"

#: kpi/serializers/v2/asset.py:165
msgid "One or many projects have been deleted already!"
msgstr "Egy vagy több projekt már törölve van!"

#: kpi/serializers/v2/asset.py:173
msgid "One or many projects are already being deleted!"
msgstr "Egy vagy több projekt törlés alatt áll!"

#: kpi/serializers/v2/asset.py:246
msgid "`action` parameter is required"
msgstr "`action` paraméter szükséges"

#: kpi/serializers/v2/asset.py:251
msgid "Supported values for `action` are: "
msgstr "Az `action` elfogadható értékei: "

#: kpi/serializers/v2/asset.py:273
msgid "Draft projects cannot be archived"
msgstr "Vázlat állapotú projektek nem archiválhatók"

#: kpi/serializers/v2/asset.py:279 kpi/serializers/v2/data.py:61
msgid "Confirmation is required"
msgstr "Megerősítés szükséges"

#: kpi/serializers/v2/asset.py:780
msgid "The property is required"
msgstr "A tulajdonság szükséges"

#: kpi/serializers/v2/asset.py:784
msgid "The property must be an array"
msgstr "A tulajdonságnak egy tömbnek kell lennie"

#: kpi/serializers/v2/asset.py:806
#, python-brace-format
msgid "Some fields are invalid, choices are: `{valid_fields}`"
msgstr "Néhány mező érvénytelen, a választási lehetőségek: `{valid_fields}`"

#: kpi/serializers/v2/asset.py:828
msgid "User cannot update current parent collection"
msgstr "A felhasználó nem frissítheti a jelenlegi szülőgyűjteményt"

#: kpi/serializers/v2/asset.py:838
#: kpi/tests/api/v2/test_api_collections.py:482
msgid "Target collection not found"
msgstr "A célgyűjtemény nem található"

#: kpi/serializers/v2/asset.py:842
#: kpi/tests/api/v2/test_api_collections.py:472
msgid "User cannot update target parent collection"
msgstr "A felhasználó nem frissítheti a cél szülőgyűjteményt"

#: kpi/serializers/v2/asset_export_settings.py:64
msgid "`export_settings` must contain all the following required keys: {}"
msgstr "Az `export_settings`-nek tartalmaznia kell a következő szükséges kulcsokat: {}"

#: kpi/serializers/v2/asset_export_settings.py:75
msgid "`export_settings` can contain only the following valid keys: {}"
msgstr "Az `export_settings` csak a következő érvényes kulcsokat tartalmazhatja: {}"

#: kpi/serializers/v2/asset_export_settings.py:87
msgid "`multiple_select` must be either {}"
msgstr "A `multiple_select` csak a következők egyike lehet: {}"

#: kpi/serializers/v2/asset_export_settings.py:94
msgid "`type` must be either {}"
msgstr "A `type` csak a következők egyike lehet: {}"

#: kpi/serializers/v2/asset_export_settings.py:104
msgid "`group_sep` must be a non-empty value"
msgstr "A `group_sep` csak nem üres érték lehet"

#: kpi/serializers/v2/asset_export_settings.py:109
msgid "`lang` for this asset must be either {}"
msgstr "A jelenlegi eszközhöz a `lang` csak a következők egyike lehet: {}"

#: kpi/serializers/v2/asset_export_settings.py:119
#: kpi/serializers/v2/export_task.py:229
msgid "Must be a JSON object"
msgstr "Egy JSON objektumnak kell lennie"

#: kpi/serializers/v2/asset_export_settings.py:134
#: kpi/serializers/v2/export_task.py:247
msgid "All values in the array must be integers"
msgstr "A tömbben minden értéknek egész számnak kell lennie"

#: kpi/serializers/v2/asset_export_settings.py:145
msgid "`fields` must be an array"
msgstr "A `fields` csak tömb lehet"

#: kpi/serializers/v2/asset_export_settings.py:149
msgid "All values in the `fields` array must be strings"
msgstr "A `fields` tömbben minden értéknek karakterláncnak kell lennie"

#: kpi/serializers/v2/asset_file.py:117
msgid "JSON is invalid"
msgstr "Érvénytelen JSON"

#: kpi/serializers/v2/asset_file.py:130
msgid "Invalid content"
msgstr "Érvénytelen tartalom"

#: kpi/serializers/v2/asset_file.py:143
msgid "No files have been submitted"
msgstr "Nem lettek beküldve fájlok"

#: kpi/serializers/v2/asset_file.py:158
msgid "File already exists"
msgstr "A fáj már létezik"

#: kpi/serializers/v2/asset_file.py:208
msgid ""
"You cannot upload media file with two different ways at the same time. "
"Please choose between binary upload, base64 or remote URL."
msgstr "Nem tölthet fel médiafájlt két különböző módon egyidőben. Kérjük, válasszon a bináris feltöltés, base64 és távoli URL között."

#: kpi/serializers/v2/asset_file.py:221
msgid "This field is required"
msgstr "A mező kitöltése kötelező"

#: kpi/serializers/v2/asset_file.py:229
msgid "`redirect_url` is required"
msgstr "A `redirect_url` szükséges"

#: kpi/serializers/v2/asset_file.py:239
msgid "`filename` is required"
msgstr "A `filename` szükséges"

#: kpi/serializers/v2/asset_file.py:253
msgid "`redirect_url` is invalid"
msgstr "A `redirect_url` érvénytelen"

#: kpi/serializers/v2/asset_file.py:287
msgid "Only `{}` MIME types are allowed"
msgstr "Csak `{}` MIME- típusok engedélyezettek"

#: kpi/serializers/v2/asset_permission_assignment.py:162
msgid "This field is required for the '{}' permission"
msgstr "Ez a mező szükséges a(z) „{}” engedélyhez"

#: kpi/serializers/v2/asset_permission_assignment.py:176
#: kpi/serializers/v2/asset_permission_assignment.py:181
#: kpi/serializers/v2/asset_permission_assignment.py:186
#: kpi/serializers/v2/asset_permission_assignment.py:492
msgid "Invalid `url`"
msgstr "Érvénytelen `url`"

#: kpi/serializers/v2/asset_permission_assignment.py:191
msgid "Invalid `filters`"
msgstr "Érvénytelen `filters`"

#: kpi/serializers/v2/asset_permission_assignment.py:208
#, python-brace-format
msgid ""
"{permission} cannot be assigned explicitly to Asset objects of this type."
msgstr "{permission} nem lehet explicit módon hozzárendelve egy ilyen típusú eszköz objektumhoz."

#: kpi/serializers/v2/asset_permission_assignment.py:286
msgid "Not a valid list."
msgstr "Nem érvényes lista."

#: kpi/serializers/v2/asset_permission_assignment.py:485
#: kpi/serializers/v2/asset_permission_assignment.py:488
msgid "Invalid hyperlink - Object does not exist."
msgstr "Érvénytelen hiperhivatkozás – Az objektum nem létezik."

#: kpi/serializers/v2/data.py:70
msgid "`submission_ids` must only contain integer values"
msgstr "`submission_ids` csak egész értéket tartalmazhat"

#: kpi/serializers/v2/data.py:75
msgid "`submission_ids` must contain at least one value"
msgstr "A `submission_ids` legalább egy értéket tartalmaznia kell"

#: kpi/serializers/v2/data.py:83
msgid "`data` is required"
msgstr "A `data` szükséges"

#: kpi/serializers/v2/data.py:91
msgid "`validation_status.uid` is required"
msgstr "A `validation_status.uid` szükséges"

#: kpi/serializers/v2/export_task.py:123
msgid "Must contain all the following required keys: {}"
msgstr "A következő szükséges kulcsok mindegyikét tartalmaznia kell: {}"

#: kpi/serializers/v2/export_task.py:137
msgid "Can contain only the following valid keys: {}"
msgstr "Csak a következő érvényes kulcsokat tartalmazhatja: {}"

#: kpi/serializers/v2/export_task.py:152 kpi/serializers/v2/export_task.py:237
msgid "Must be an array"
msgstr "Tömbnek kell lennie"

#: kpi/serializers/v2/export_task.py:159
msgid "All values in the array must be strings"
msgstr "A tömbben minden értéknek karakterláncnak kell lennie"

#: kpi/serializers/v2/export_task.py:169
msgid "Must be a non-empty value"
msgstr "Egy nem üres értéknek kell lennie"

#: kpi/serializers/v2/export_task.py:182
msgid "For this asset must be either {}"
msgstr "Ehhez az eszközhöz csak a következők egyike lehet: {}"

#: kpi/serializers/v2/export_task.py:194 kpi/serializers/v2/export_task.py:258
msgid "Must be either {}"
msgstr "Csak a következők egyike lehet: {}"

#: kpi/serializers/v2/export_task.py:204
msgid "The asset must be deployed."
msgstr "Az eszközt élesíteni kell."

#: kpi/serializers/v2/export_task.py:221
msgid "The export name must be a string."
msgstr "Az exportnévnek karakterláncnak kell lennie."

#: kpi/serializers/v2/paired_data.py:87
msgid "Source cannot be changed"
msgstr "A forrás nem változtatható meg"

#: kpi/serializers/v2/paired_data.py:93
#, python-brace-format
msgid "Data sharing for `{source_uid}` is not enabled"
msgstr "Adatmegosztás `{source_uid}` esetében nincs engedélyezve"

#: kpi/serializers/v2/paired_data.py:105
#, python-brace-format
msgid "Pairing data with `{source_uid}` is not allowed"
msgstr "`{source_uid}` nem párosítható adathoz"

#: kpi/serializers/v2/paired_data.py:110
#, python-brace-format
msgid "Source `{source}` is already paired"
msgstr "A(z) `{source}` forrás már párosítva van"

#: kpi/serializers/v2/paired_data.py:152
#, python-brace-format
msgid "Some fields are invalid, choices are: `{source_fields}`"
msgstr "Néhány mező érvénytelen, a választási lehetőségek: `{source_fields}`"

#: kpi/serializers/v2/paired_data.py:177
msgid "Only letters, numbers and `-` are allowed"
msgstr "Csak betűk, számok és `-` engedélyezettek"

#: kpi/serializers/v2/paired_data.py:184
msgid "Extension must be `xml`"
msgstr "A kiterjesztésnek `xml`-nek kell lennie"

#: kpi/serializers/v2/paired_data.py:217
#, python-brace-format
msgid "`{basename}` is already used"
msgstr "`{basename}` már használatban van"

#: kpi/serializers/v2/user_asset_subscription.py:47
#, python-brace-format
msgid "Invalid asset type. Only `{asset_type}` is allowed"
msgstr "Érvénytelen eszköztípus. Csak `{asset_type}` engedélyezett"

#: kpi/templates/base.html:37
msgid "Terms and Conditions"
msgstr "Felhasználási feltételek"

#: kpi/templates/base.html:40
msgid "Privacy Policy"
msgstr "Adatvédelmi szabályzat"

#: kpi/templates/modern_browsers.html:3 kpi/templates/modern_browsers.html:5
msgid "Modern Browsers"
msgstr "Modern böngészők"

#: kpi/templates/modern_browsers.html:8
msgid ""
"Ouch! Sorry, you were probably sent here because your browser is outdated or"
" perhaps it has its key features disabled by using \"private browsing\" "
"mode."
msgstr "Jaj! Sajnáljuk, valószínűleg azért került ide, mert elavult a böngészője, esetleg valamely szűkséges funkciója ki van kapcsolva a „privát böngészés” üzemmód miatt."

#: kpi/templates/modern_browsers.html:12
msgid ""
"If you are using an outdated browser, we recommend upgrading to a recent "
"version of one of the following excellent modern browsers:"
msgstr "Amennyiben elavult böngészőt használ, javasoljuk, hogy váltson az alábbi kiváló modern böngészők valamelyikének újabb verziójára:"

#: kpi/templates/modern_browsers.html:17
msgid "Firefox download page"
msgstr "Firefox letöltési oldala"

#: kpi/templates/modern_browsers.html:18
msgid "Firefox"
msgstr "Firefox"

#: kpi/templates/modern_browsers.html:22
msgid "Chrome download page"
msgstr "Chrome letöltési oldala"

#: kpi/templates/modern_browsers.html:23
msgid "Chrome"
msgstr "Chrome"

#: kpi/templates/modern_browsers.html:27
msgid "Edge download page"
msgstr "Edge letöltési oldala"

#: kpi/templates/modern_browsers.html:28
msgid "Edge"
msgstr "Edge"

#: kpi/templates/modern_browsers.html:32
msgid "Opera download page"
msgstr "Opera letöltési oldala"

#: kpi/templates/modern_browsers.html:33
msgid "Opera"
msgstr "Opera"

#: kpi/tests/test_registration.py:70
msgid "This email domain is not allowed to create an account"
msgstr "Ezzel az e-mail tartománnyal nem lehet fiókot létrehozni"

#: kpi/utils/two_database_configuration_checker.py:41
msgid "Exactly two databases must be configured"
msgstr "Pontosan két adatbázist kell konfigurálni"

#: kpi/utils/two_database_configuration_checker.py:42
msgid ""
"KPI and KoBoCAT must each have their own databases. Configure the KPI "
"database as 'default' and the KoBoCAT database as 'kobocat'."
msgstr "A KPI-nak és a KoBoCAT-nek is saját adatbázissal kell rendelkeznie. A KPI adatbázist „default” néven, a KoBoCAT adatbázist pedig „kobocat” néven kell konfigurálni."

#: kpi/utils/two_database_configuration_checker.py:63
msgid "KPI may not share a database with KoBoCAT"
msgstr "A KPI és a KoBoCAT nem használhatnak közös adatbázist"

#: kpi/utils/two_database_configuration_checker.py:64
msgid "KPI and KoBoCAT must each have their own databases."
msgstr "KPI-nak and KoBoCAT-nek is saját adatbázissal kell rendelkeznie."

#: kpi/utils/two_database_configuration_checker.py:133
msgid "Incomplete migration from shared-database installation"
msgstr "Hiányos áttelepítés a közös adatbázis telepítésből"

#: kpi/utils/two_database_configuration_checker.py:134
msgid ""
"The KoBoCAT database was originally shared by KPI, but the KPI tables were "
"not copied from that shared database to the new, KPI-only database."
msgstr "A KoBoCAT-adatbázis eredetileg meg volt osztva a KPI-vel, de a KPI-táblákat nem másolták át abból a közösen használt adatbázisból az új, csak KPI által használt adatbázisba."

#: kpi/views/current_user.py:106
msgid "Invalid confirmation"
msgstr "Érvénytelen megerősítés"

#: kpi/views/v2/asset_export_settings.py:228
msgid "Only the following formats are available: ##format list##"
msgstr "Csak a következő formátumok elérhetők: ##format list##"

#: kpi/views/v2/asset_permission_assignment.py:193
msgid "Source and destination objects don't seem to have the same type"
msgstr "A forrás- és a célobjektum típusa nem egyezik meg"

#: kpi/views/v2/asset_permission_assignment.py:220
msgid "Owner's permissions cannot be deleted"
msgstr "A tulajdonos engedélyei nem törölhetők"

#: kpi/views/v2/attachment.py:73
msgid "`xpath` query parameter is required"
msgstr "`xpath` lekérdezési paraméter szükséges"

#: kpi/views/v2/attachment.py:94
msgid "Invalid XPath syntax"
msgstr "Érvénytelen XPath-szintaxis"

#: kpi/views/v2/attachment.py:98
msgid "The path could not be found in the submission"
msgstr "Az útvonal nem található az űrlapválaszban"

#: kpi/views/v2/attachment.py:107
msgid "The error occurred during conversion"
msgstr "A hiba a konverzió során lépett fel"

#: kpi/views/v2/attachment.py:111
msgid "Conversion is not supported for {}"
msgstr "A konverzió nem támogatott a(z) {} esetén"

#: kpi/views/v2/data.py:317
msgid "The specified asset has not been deployed"
msgstr "A megadott eszköz nem lett élesítve"

#: kpi/views/v2/data.py:621
msgid "A positive integer is required"
msgstr "Pozitív egész szám szükséges"
